ext/mysqli/tests/functions/mysqli_character_set_name.phpt

```diff
@@ -20,6 +20,6 @@ require_once('skipifconnectfailure.inc');
     mysqli_close($link);
     print "done!";
 ?>
---EXPECTF--
-string(%d) "%s"
+--EXPECT--
+string(6) "utf8mb"
```

Comment:
We can't do this. We don't actually know what the charset will be until we set it. We don't set the charset anywhere, so it could be anything. I also do not understand why on earth are we using `substr` here. 
I'd suggest to rewrite this test and call `set_charset` twice with different values. After each call check if the charset actually changed.

Reply:
Should I delete this as we have ``ext/mysqli/tests/functions/mysqli_set_charset.phpt`` already?

Reply:
We also have `ext\mysqli\tests\mysqli_character_set_name.phpt` so I think it's safe to remove this one.